### DISCUSSION TITLE
Error: You have provided fees that we cannot attribute to OpenSea or the collection, in klaytn chain

### DISCUSSION THREAD

**Discussion by suropark** [ProjectOpenSea/opensea-js#1409] [Q&A]
hi, is there anyone knows how to handle this error in klaytn chain

```
"You have provided fees that we cannot attribute to OpenSea or the collection"
```

chain  : klaytn 

```
      const offer = await this.sdk.createOffer({
        asset: {
          tokenId,
          tokenAddress,
        },
        accountAddress: accountAddress,
        startAmount: price,
        expirationTime: (Math.round(Date.now() / 1000) + duration).toString(),
      });

```

all parameters are filled correctly

**Comment by suropark** [comment]
in opensea page, fee recipient which is one of consideration is `0x8De9C5A032463C561423387a9648c5C7BCC5BC90` it succeed.

ex)
```
       {
            "itemType": "0",
            "token": "0x0000000000000000000000000000000000000000",
            "identifierOrCriteria": "0",
            "startAmount": "125000000000000000",
            "endAmount": "125000000000000000",
            "recipient": "0x8De9C5A032463C561423387a9648c5C7BCC5BC90"
        },
```

but using sdk,

```
        {
            "itemType": "0",
            "token": "0x0000000000000000000000000000000000000000",
            "identifierOrCriteria": "0",
            "startAmount": "125000000000000000",
            "endAmount": "125000000000000000",
            "recipient": "0x0000a26b00c1f0df003000390027140000faa719"
        },
```

do you guys think this can be problem ??



i tried 
chain : klaytn
erc721 address: 0x49d1a936D85de42eF72739955c6e74856a8068B5 (klaytn-localstitch)
tokenId : 542
offerPrice: 5

**Comment by ryanio** [comment]
`0x0000a26b00c1f0df003000390027140000faa719` is the OpenSea fee wallet.

What's the full consideration list from the sdk? It should be: the seller, os fee wallet for 2.5%, and any "required" creator fees from the Get Collection API "fees" response

**Reply by AsliddinBuriev** [reply]
Hi thank you for your response!
Here is the full list of consideration: 
[
    {
        "itemType": 2,
        "token": "0x5bebd7cdeb76b37102a2e4b26093209e3a4b5750",
        "identifierOrCriteria": "275",
        "startAmount": "1",
        "endAmount": "1",
        "recipient": "0x8aaf553d422c2200c25b64bd652f14fc041571f5"
    },
    {
        "itemType": 1,
        "token": "0xfd844c2fca5e595004b17615f891620d1cb9bbb2",
        "identifierOrCriteria": "0",
        "startAmount": "25000000000000000",
        "endAmount": "25000000000000000",
        "recipient": "0x0000a26b00c1f0df003000390027140000faa719"
    },
    {
        "itemType": 1,
        "token": "0xfd844c2fca5e595004b17615f891620d1cb9bbb2",
        "identifierOrCriteria": "0",
        "startAmount": "50000000000000000",
        "endAmount": "50000000000000000",
        "recipient": "0x1f2d6594a04f5f6d0daaabaec192c56358138a09"
    }
]

**Reply by AsliddinBuriev** [reply]
the response was: 
{
    "errors": [
        "You have provided fees that we cannot attribute to OpenSea or the collection"
    ]
}

**Reply by AsliddinBuriev** [reply]
0x8aaf553d422c2200c25b64bd652f14fc041571f5 is my wallet address which I am sending offer from.

**Comment by suropark** [comment]
@ryanio hi,

when i send offer or create listing using opensea directly, 
fee recipient in consideration array is different. 

In opensea webpage

`0x8De9C5A032463C561423387a9648c5C7BCC5BC90` : it works!
`0x0000a26b00c1f0df003000390027140000faa719` : it fails with error `"You have provided fees that we cannot attribute to OpenSea or the collection"`

plz check this out.

full consideration is below.

ex) create listing
In opensea webpage

```
    "consideration": [
        {
            "itemType": "0",
            "token": "0x0000000000000000000000000000000000000000",
            "identifierOrCriteria": "0",
            "startAmount": "4625000000000000000",
            "endAmount": "4625000000000000000",
            "recipient": "0x5B78ace197872A4C90bb137D0643aA3755DBC1A0"
        },
        {
            "itemType": "0",
            "token": "0x0000000000000000000000000000000000000000",
            "identifierOrCriteria": "0",
            "startAmount": "125000000000000000",
            "endAmount": "125000000000000000",
            "recipient": "0x8De9C5A032463C561423387a9648c5C7BCC5BC90" // difference
        },
        {
            "itemType": "0",
            "token": "0x0000000000000000000000000000000000000000",
            "identifierOrCriteria": "0",
            "startAmount": "250000000000000000",
            "endAmount": "250000000000000000",
            "recipient": "0x1F2D6594A04F5f6d0DAaabAEC192c56358138A09"
        }
    ],
```


using opensea-js

```
  "consideration": [
  {
    "itemType": 0,
    "token": "0x0000000000000000000000000000000000000000",
    "identifierOrCriteria": "0",
    "startAmount": "4625000000000000000",
    "endAmount": "4625000000000000000",
    "recipient": "0x5b78ace197872a4c90bb137d0643aa3755dbc1a0"
  },
  {
    "itemType": 0,
    "token": "0x0000000000000000000000000000000000000000",
    "identifierOrCriteria": "0",
    "startAmount": "125000000000000000",
    "endAmount": "125000000000000000",
    "recipient": "0x0000a26b00c1f0df003000390027140000faa719" // difference
  },
  {
    "itemType": 0,
    "token": "0x0000000000000000000000000000000000000000",
    "identifierOrCriteria": "0",
    "startAmount": "250000000000000000",
    "endAmount": "250000000000000000",
    "recipient": "0x1f2d6594a04f5f6d0daaabaec192c56358138a09"
  }
]
```
